site/training/training-for-model-validators.qmd

```diff
@@ -0,0 +1,359 @@
+# Finding your way around ValidMind
+
+## {background-iframe="https://app.prod.validmind.ai" data-preload="yes"} 
+
+:::: {.fr .mv5 .pa5 .bg-light-pink .ba .b--black-20 .bw1 .br3 .near-black .shadow-5}
+**This is the {{< var vm_platform >}}.**
+
+From here, you have access to: 
+
+::: {.columns .f3}
+::: {.column width="50%" .bl .pl3}
+- Model inventory
+- Documentation & test results
+:::
+
+::: {.column width="50%" .pl3}
+- Validation reports
+- Model findings
+- Reports 
+:::
+:::
+
+Try it yourself on the next page. {{< fa hand-point-right >}}
+::::
+
+## {background-iframe="https://app.prod.validmind.ai/model-inventory" background-interactive="true" data-preload="yes"}
+
+::: {.fr .f4 .mv7 .nr5 .pa4 .bg-light-pink .ba .b--black-20 .bw1 .br3 .near-black.}
+From the **Model Inventory**, <br>you access:
+
+- Model documentation 
+- Validation reports
+- Getting started for <br>developers
+
+Explore some models, then <br>click [{{< fa chevron-right >}}]() to continue.
+:::
+
+## {background-iframe="https://app.prod.validmind.ai/model-findings" background-interactive="true" data-preload="yes"}
+
+:::: {.fr .f4 .mv7 .nr5 .pa4 .bg-light-pink .ba .b--black-20 .bw1 .br3 .near-black .shadow-5}
+From **Model Findings**, <br>you track:
+
+- Major or minor issues
+- Deficiencies
+- Model limitations
+- Robustness concerns
+- And more!
+
+Explore existing findings, <br>then click [{{< fa chevron-right >}}]() to continue.
+::::
+
+## {background-iframe="https://app.prod.validmind.ai/reports" background-interactive="true" data-preload="yes"}
+
+
+:::: {.fr .f4 .mv7 .nr5 .pa4 .bg-light-pink .ba .b--black-20 .bw1 .br3 .near-black .shadow-5}
+Under **Reports**, you can <br>find:
+
+- Executive summaries
+- Model tracking
+- Risk assessments
+- Compliance review
+- Remediation tracking
+
+Explore existing reports, <br>then click [{{< fa chevron-right >}}]() to continue.
+::::
+
+# 1. Review documentation
+
+## {.scrollable}
+
+:::: {.columns}
+::: {.column width="30%"  .pr4}
+Verify model documentation is complete and complies with regulatory standards by cross-checking against your guidelines. 
+:::
+
+::: {.column width="70%" .bl .pl4 .f3}
+### Review model documentation
+
+{{< include ../guide/_review-model-documentation.qmd >}}
+
+<form method="get" action="../guide/review-model-documentation.html" target="_blank">
+   <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 24px; padding: 6.25px 12.5px; margin-top: 10px; margin-bottom: 20px;">Learn more ...</button>
+</form>
+:::
+::::
+
+Try it yourself on the next page. {{< fa hand-point-right >}}
+
+## {background-iframe="https://app.prod.validmind.ai" background-interactive="true" data-preload="yes"}
+
+::: {.absolute bottom=15 .w-100 .f4 .tc .pl4 .bg-light-pink .ba .b--black-20 .bw1 .br3 .near-black .shadow-5}
+**Try it:** Review the documentation for the **[Quickstart] Customer Churn Model**. | Click [{{< fa chevron-right >}}]() to continue.
```

Comment:
> I notice we are doing a little hack here where we use an empty link to change the colour of the chevron to pink... surely there is a better way? 😅

The links will never work once you start interacting with the product UI as the input focus gets shifted to the background layer. If you know how to work around this, I'd love to hear about it!

> LOBAL COMMENT: I think the | divider is a bit inelegant too

It was a late addition to deal with the focus shifting. I don't think the raised elipsis necessarily looks any cleaner, but the next version of this might move that out from the text into another part of the callout with some extra CSS.

Reply:
I ended up just removing the text altogether. Between the nonfunctional link and the need to go taller to make this look OK, it's just too much of a pain right now. There's a lingering navigation issue with the different layers you interact with the product UI layuer regardless, so might be good to take a step back and revisit what our preferred navigation should be.